dotnet/AdapterSdkSchema.sln

```diff
@@ -18,7 +18,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MicroFocus.FAS.AdapterSdkSc
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MicroFocus.FAS.AdapterSdkSchema.Generator", "src\MicroFocus.FAS.AdapterSdkSchema.Generator\MicroFocus.FAS.AdapterSdkSchema.Generator.csproj", "{9F484E11-38C6-4765-9D92-F32BBC27E9A0}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MicroFocus.FAS.AdapterSdkSchema.Generator.Tests", "tests\MicroFocus.FAS.AdapterSdkSchema.Generator.Tests\MicroFocus.FAS.AdapterSdkSchema.Generator.Tests.csproj", "{348DFA4B-7117-4534-967A-F4D58E3AEDC5}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MicroFocus.FAS.AdapterSdkSchema.Generator.Tests", "tests\MicroFocus.FAS.AdapterSdkSchema.Generator.Tests\MicroFocus.FAS.AdapterSdkSchema.Generator.Tests.csproj", "{348DFA4B-7117-4534-967A-F4D58E3AEDC5}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MicroFocus.FAS.AdapterSdkSchema.JsonBuilders.System", "src\MicroFocus.FAS.AdapterSdkSchema.JsonBuilders.System\MicroFocus.FAS.AdapterSdkSchema.JsonBuilders.System.csproj", "{F866CD3C-CF32-4D60-A9C3-C1B319080B11}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MicroFocus.FAS.AdapterSdkSchema.SchemaObjectBuilders", "src\MicroFocus.FAS.AdapterSdkSchema.SchemaObjectBuilders\MicroFocus.FAS.AdapterSdkSchema.SchemaObjectBuilders.csproj", "{15CFB8B2-B957-4CBD-B084-FDCB73023893}"
```

Comment:
I think maybe we should fold this project into the main one to align with the Java version.